internal/provider/resource_gitlab_project.go

```diff
@@ -1269,12 +1269,7 @@ func resourceGitlabProjectRead(ctx context.Context, d *schema.ResourceData, meta
 func resourceGitlabProjectUpdate(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
 	client := meta.(*gitlab.Client)
 
-	// Always send the name field, to satisfy the requirement of having one
```

Comment:
🚧 **Suggestion:** Can you please remove these changes in the project resource from this PR?

Reply:
Ups, mix up with branches. Removing

Reply:
Removed